README.md

```diff
@@ -102,6 +102,25 @@ the secrets stored in temp files and in the Python process environment are gone.
 
     This flag can be useful for when you have secrets that you don't need access to for development. For example API keys for monitoring tools. This flag can be used multiple times.
 
+* `-e, --environment` Specify section (environment) to parse from secret YAML
+
+    This flag specifies which specific environment/section to parse from the secrets YAML file (or string). In addition, it will also enable the usage of a `common` section which will be inherited by other sections/environments. In other words, if your `secrets.yaml` looks something like this:
+
+```yaml
+common:
```

Comment:
`common` makes more semantic sense to me, since it's not a real environment. That said, maybe some will just one one environment, then `default` would make more sense.

We're not using secrets.yml like this internally at Conjur (rather we set naming schemes for secrets and interpolate them with `-D`), so open to suggestions.